security/src/main/java/uk/ac/ox/ndph/mts/security/authentication/AADWebSecurityConfig.java

```diff
@@ -43,6 +43,8 @@ protected void configure(HttpSecurity http) throws Exception {
         // disable csrf because we are using another token mechanism
         http.csrf().disable(); //NOSONAR
 
+        http.cors(); // See https://www.baeldung.com/spring-cors
```

Comment:
@LizaShak and @mdp0 I want to draw your attention this since it's a change to the core security package.   It's needed for story 477 but it's really a wider issue for any UI stories.